lightning/src/ln/channel.rs

```diff
@@ -2133,6 +2135,14 @@ impl<Signer: Sign> Channel<Signer> {
 		)
 	}
 
+	/// Get our total balance in msat.
+	/// This is the amount that would go to us if we close the channel, ignoring any on-chain fees.
+	/// See also [`Channel::get_inbound_outbound_available_balance_msat`]
+	pub fn get_balance_msat(&self) -> u64 {
+		self.value_to_self_msat
```

Comment:
I don't know the use-case you have in mind, though why not subtracting the fees if it's the funder's balance to get the most-correct on-chain claimable amount ?

Reply:
This is just the balance to display to the user as "this is how many sats are in your Lightning wallet".  The existing getter (just below) is not suitable because it is lower by about 1% due to the reserve.

The actual fee will depend on when the channel is closed, which could be at some arbitrary point in the future.  If the user is patient, the fee could be pretty low.

Also, it would be strange if your displayed balance changed without any user action.

Reply:
FYI, from my experience people are actually more confused by not being able to spend whole balance. And if they know about the reserve it may become a guess work to find out how much they can actually spend. The UIs should probably display spendable balance and the reserve.